staging/src/k8s.io/apiserver/pkg/storage/testing/watcher_tests.go

```diff
@@ -1319,15 +1319,26 @@ func RunWatchSemantics(ctx context.Context, t *testing.T, store storage.Interfac
 				return []watch.Event{{Type: watch.Added, Object: createdPodsAfterWatch[0]}}
 			},
 		},
+		{
+			name:                               "sendInitialEvents=false, RV=17",
+			sendInitialEvents:                  &falseVal,
+			resourceVersion:                    "17",
```

Comment:
This highly depends on the test-case order.
Can we instead add sth like `useCurrentRV` and compute RV in the test itself by calling consistent list (with RV unset)?